docker/files/run_examples.sh

```diff
@@ -12,10 +12,16 @@ mkdir -p ${HOME}/.nipype ${WORKDIR}/logs/example_${example_id} ${WORKDIR}/tests
 echo "[logging]" > ${HOME}/.nipype/nipype.cfg
 echo "workflow_level = DEBUG" >> ${HOME}/.nipype/nipype.cfg
 echo "interface_level = DEBUG" >> ${HOME}/.nipype/nipype.cfg
-echo "filemanip_level = DEBUG" >> ${HOME}/.nipype/nipype.cfg
+echo "utils_level = DEBUG" >> ${HOME}/.nipype/nipype.cfg
 echo "log_to_file = true" >> ${HOME}/.nipype/nipype.cfg
 echo "log_directory = ${WORKDIR}/logs/example_${example_id}" >> ${HOME}/.nipype/nipype.cfg
 
+if [[ "${NIPYPE_RESOURCE_MONITOR:-0}" == "1" ]]; then
+    echo '[execution]' >> ${HOME}/.nipype/nipype.cfg
```

Comment:
just a note that `.nipype` is no longer created by default. so nipype doesn't touch anything when run.

Reply:
mmm what you mean? this is currently working out...

Reply:
if it's working great - just that in the past nipype would create this directory by default, now it doesn't.

Reply:
if it still reads it, then it works :)